src/Dfe.EarlyYearsQualification.Web/Views/QualificationDetails/Index.cshtml

```diff
@@ -49,6 +50,20 @@
                     @Model.FromWhichYear
                 </dd>
             </div>
+            @if (Model.AdditionalRequirementAnswers != null)
+            {
+                for (var count = 0; count < Model.AdditionalRequirementAnswers.Count; count++)
+                {
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key" id="additional-requirement-@count-label">
+                            @Model.AdditionalRequirementAnswers[count].Question

```

Comment:
The text displayed here should be the Confirmation Statement instead of the question I believe. So instead of it reading 'Does the qualification name include Early Years Educator?', it should be 'Qualification name includes Early Years Educator'